thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/rootcause/impl/HolidayEventsLoader.java

```diff
@@ -0,0 +1,142 @@
+package com.linkedin.thirdeye.rootcause.impl;
```

Comment:
I'd move this to the existing events package "com.linkedin.thirdeye.anomaly.events"